apps/console/src/features/applications/components/forms/advanced-configurations-form.tsx

```diff
@@ -241,7 +243,7 @@ export const AdvancedConfigurationsForm: FunctionComponent<AdvancedConfiguration
                 data-testid={ `${ testId }-sass-checkbox` }
                 data-componentid={ `${ testId }-sass-checkbox` }
                 hint={ t("console:develop.features.applications.forms.advancedConfig.fields.saas.hint") }
-                hidden={ !applicationConfig.advancedConfigurations.showSaaS }
+                hidden={ !legacyAuthzRuntime || !applicationConfig.advancedConfigurations.showSaaS }
```

Comment:
Should we put this under `legacy mode` configs? @SujanSanjula96 ?

Reply:
If there's no incremental migration, then it's okay to have `legacyAuthzRuntime`

Reply:
+1 to put this as a separate config under legacy mode configs.
Reference PRs:
https://github.com/wso2/carbon-identity-framework/pull/5266
https://github.com/wso2/carbon-identity-framework/pull/5282

Reply:
Added new config under `legacy mode`